### DISCUSSION TITLE
Tracing-log with custom env-logger format

### DISCUSSION THREAD

**Discussion by DianaSensei** [tokio-rs/tracing#2267] [Q&A]
Hi, i'm using env-logger with custom format (for build traceId, spanId purpose). Now i want to use tracing and found that "tracing-log" has feature "env-logger" but cannot change or modify the format of env-logger. Any one have a solution for my issue?

**Comment by davidbarsky** [comment]
If your application is already `tracing` macros (or a mixture of `tracing` and `log`), then I suggest using [`tracing_subscriber::fmt`](https://docs.rs/tracing-subscriber/latest/tracing_subscriber/fmt/index.html), which provides an `env-logger`-like experience.
